test/tests/Xsc/XscClient.spec.ts

```diff
@@ -0,0 +1,52 @@
+import nock from 'nock';
+import { XscClient } from '../../../src/Xsc/XscClient';
+import { TestUtils } from '../../TestUtils';
+import { XscSystemClient } from '../../../src/Xsc/XscSystemClient';
+import { XscEventClient } from '../../../src/Xsc/XscEventClient';
+
+const SERVER_URL: string = 'http://localhost:8000';
+
+beforeAll(() => {
+    nock.disableNetConnect();
+    nock.enableNetConnect(SERVER_URL);
+});
+
+afterAll(() => {
+    nock.cleanAll();
+    nock.enableNetConnect();
+});
+
+describe('Xsc clients tests', () => {
+    test('Client initialization', () => {
+        const client: XscClient = new XscClient({ serverUrl: SERVER_URL });
+        expect(client).toBeInstanceOf(XscClient);
+    });
+    test('Client w/o url', () => {
```

Comment:
commented somewhere else, but think we should add a test for a case there's no credentials

Reply:
creds are optional here